internal/cmd/local/local/cmd.go

```diff
@@ -294,6 +297,81 @@ func (c *Command) Install(ctx context.Context, user, pass string) error {
 	return nil
 }
 
+func (c *Command) watchEvents(ctx context.Context) {
+	watcher, err := c.k8s.EventsWatch(ctx, airbyteNamespace)
+	if err != nil {
+		pterm.Warning.Printfln("Unable to watch airbyte events\n  %s", err)
```

Comment:
NIT: The phrasing here is a little misleading in that these are K8s events. Not sure if it matters much, but if the perception might be that "Airbyte failed on something" vs "Something went wrong with k8s" it doesn't hurt our optics as much.